### PR TITLE
feat: add paid webhook

### PR DESCRIPTION
**Pull request by wesleymatosdev**
- feat: add BOOKING_PAID on enums
- feat: trigger the BOOKING_PAID webhook

## What does this PR do?

Adds the `BOOKING_PAID` webhook from #7299 

edit: Also creates `BOOKING_REQUESTED` and `BOOKING_REJECTED` on the database

## Type of change

- New feature (non-breaking change which adds functionality)
- This change requires a documentation update

## How should this be tested?

- [x] We need to ensure that the webhook is really being triggered when the charge of the payment occurs

## Checklist

- I haven't checked if my PR needs changes to the documentation
- I haven't added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by alishaz-polymath — CHANGES_REQUESTED** [review]
Two major things:
1. It's missing a migration. There's change in the DB (prisma/schema.prisma) so it needs to have a migration as well, and please ensure that the migration doesn't destroy the existing webhooks.
2. Ideally we'd want to have as little migrations as possible, so please add the `BOOKING_REQUESTED` and `BOOKING_REJECTED` in the same migration, hence the respective change in `constants.ts` and `schema.prisma`. 
We can have them function with a follow up PR, but we definitely want them in the DB in one go, to minimize subsequent migrations
Thanks for the PR 🙌

**Comment by ologbonowiwi** [comment]
`BOOKING_REQUESTED` and `BOOKING_REJECTED` were added.

How do I generate the migration to create this stuff on the database?

**Comment by ologbonowiwi** [comment]
All webhooks added.

Can you review it again, @alishaz-polymath?

File: packages/features/webhooks/lib/constants.ts [hunk 1]
```diff
@@ -6,7 +6,10 @@ export const WEBHOOK_TRIGGER_EVENTS_GROUPED_BY_APP = {
   core: [
     WebhookTriggerEvents.BOOKING_CANCELLED,
     WebhookTriggerEvents.BOOKING_CREATED,
+    WebhookTriggerEvents.BOOKING_REJECTED,
+    WebhookTriggerEvents.BOOKING_REQUESTED,
```
Comment (alishaz-polymath) [inline packages/features/webhooks/lib/constants.ts:10]:
```suggestion

```

File: packages/prisma/schema.prisma [hunk 2]
```diff
@@ -494,6 +494,9 @@ enum PaymentOption {
 
 enum WebhookTriggerEvents {
   BOOKING_CREATED
+  BOOKING_PAID
+  BOOKING_REJECTED
+  BOOKING_REQUESTED
```
Comment (alishaz-polymath) [inline packages/prisma/schema.prisma:499]:
```suggestion

```

File: packages/prisma/migrations/20230513235419_add_booking_webhooks/migration.sql [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+-- This migration adds more than one value to an enum.
+-- With PostgreSQL versions 11 and earlier, this is not possible
+-- in a single migration. This can be worked around by creating
+-- multiple migrations, each migration adding only one value to
+-- the enum.
+
+
+ALTER TYPE "WebhookTriggerEvents" ADD VALUE 'BOOKING_PAID';
+ALTER TYPE "WebhookTriggerEvents" ADD VALUE 'BOOKING_REJECTED';
+ALTER TYPE "WebhookTriggerEvents" ADD VALUE 'BOOKING_REQUESTED';
```
Comment (alishaz-polymath) [inline packages/prisma/migrations/20230513235419_add_booking_webhooks/migration.sql:11]:
```suggestion

```

**Review by alishaz-polymath — CHANGES_REQUESTED** [review]
Please make sure to accommodate changes made in the PR #8884 so as to not repeat the same modifications and duplicate them. 🙏

**Comment by ologbonowiwi** [comment]
Changes applied, thanks @alishaz-polymath

**Review by zomars — APPROVED** [review]
LGTM

**Comment by ologbonowiwi** [comment]
this will be merged?

changes already had been addressed @PeerRich @alishaz-polymath @zomars

**Review by alishaz-polymath — APPROVED** [review]
Let's go! 🚀 
Thanks a lot for your contribution 🙏

**Pull request merged**